contracts/TokenBridgeBaseUpgradable.sol

```diff
@@ -36,4 +39,6 @@ abstract contract TokenBridgeBase is NonblockingLzApp, ReentrancyGuard {
 
     /// @dev Overrides the renounce ownership logic inherited from openZeppelin `Ownable`
     function renounceOwnership() public override onlyOwner {}
```

Comment:
should this function be empty?

Reply:
They have disabled this functionality so that Bridge is owned by someone, as it is required at many places eg adding new tokens and, withdrawing fee,s etc